docs/diagrams/StorageClassDiagram.puml

```diff
@@ -6,22 +6,22 @@ skinparam classBackgroundColor STORAGE_COLOR
 
 package Storage as StoragePackage {
 
-package "UserPrefs Storage" #F4F6F6{
-Class "<<interface>>\nUserPrefsStorage" as UserPrefsStorage
-Class JsonUserPrefsStorage
-}
-
-Class "<<interface>>\nStorage" as Storage
-Class StorageManager
-
-package "AddressBook Storage" #F4F6F6{
-Class "<<interface>>\nAddressBookStorage" as AddressBookStorage
-Class JsonAddressBookStorage
-Class JsonSerializableAddressBook
-Class JsonAdaptedPerson
-Class JsonAdaptedTag
-}
-
+    package "UserPrefs Storage" #F4F6F6 {
+        Class "<<interface>>\nUserPrefsStorage" as UserPrefsStorage
+        Class JsonUserPrefsStorage
+    }
+
+    Class "<<interface>>\nStorage" as Storage
+    Class StorageManager
+
+    package "AddressBook Storage" #F4F6F6 {
+        Class "<<interface>>\nAddressBookStorage" as AddressBookStorage
+        Class JsonAddressBookStorage
+        Class JsonSerializableAddressBook
+        Class JsonAdaptedPerson
+        Class JsonAdaptedWedding
+        Class JsonAdaptedRole
```

Comment:
![image](https://github.com/user-attachments/assets/ca7fe785-ca34-4aa7-999e-88e4ae0b645b)
jsonAdaptedRole is not used anymore, cuz its just a normal class now instead of a Set<>